deploy/kubernetes/snapshot/validation-webhook/README.md

```diff
@@ -0,0 +1,62 @@
+# Snapshot validation webhook
+For a better understanding of webhooks and validation webhook in kubernetes, please refer  [this](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+In simple terms, the snapshot validation webhook when deployed, validates the input provided for creating/updating snapshot as described below:
+1. Kubernetes will return error when you try to create a snapshot without volumeSnapshotClassName. Without the webhook, no error is returned, but snapshot won't get created.
+2. Kubernetes will return error when you try to update the snapshot with a different pvc name. Without the web hook, no error is returned, but snapshot config will be updated to new pvc given, but it will still be a snapshot of the previous pvc.
+
+To understand more about snapshot validation web hook and it's benefits, refer [snapshot validation webhook]( https://github.com/kubernetes-csi/external-snapshotter/tree/v6.0.1#validating-webhook)
+
+## Deploying snapshot validation webhook
+### Deploying webhook without cert manager
+**Note:** This is not considered to be a production ready method to deploy the certificates. This is only one of many ways to deploy the certificates, it is your responsibility to ensure the security of your cluster. TLS certificates and private keys should be handled with care and you may not want to keep them in plain Kubernetes secrets.
+
+The steps to deploy this webhook is already provided [here](https://github.com/kubernetes-csi/external-snapshotter/blob/v6.0.1/deploy/kubernetes/webhook-example/README.md#how-to-deploy-the-webhook), but the same might not work on your cluster, hence follow these steps (these are the same steps provided in this [READme](https://github.com/kubernetes-csi/external-snapshotter/blob/v6.0.1/deploy/kubernetes/webhook-example/README.md#how-to-deploy-the-webhook) but with some modifications and more details).
+1. Clone [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter) repo. The following steps should be run from the same external-snapshotter folder.
+2. Run the `create-cert.sh` script. More details about the script are mentioned below.
+```
+./deploy/kubernetes/webhook-example/create-cert.sh --service snapshot-validation-service --secret snapshot-validation-secret --namespace default
+```
+- The script takes 3 arguments - `service` which is name of the k8s service for the webhook, `secret` which is the name of the k8s secret which will hold RSA key and certificate, `namespace` which is the namespace where the pods, secret and service related to this webhook is deployed. It these arguments are not provided, the default values are `service - admission-webhook-example-svc`, `secret - admission-webhook-example-certs`, `namespace - default`.
```

Comment:
nit, `If these arguments are not ...` instead of `It these arguments are not ...`